### ISSUE TITLE
Please add connector to mattermost

### ISSUE THREAD

**Issue by mihai-satmarean** [RasaHQ/helpdesk-assistant#59]
Please add connector to mattermost and example configuration.
Since I am a noob, I can help if properly guided.

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @dakshvar22 will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by HotThoughts** [comment]
Would this help?
https://rasa.com/docs/rasa/connectors/mattermost/

**Comment by mihai-satmarean** [comment]
Thanks @HotThoughts I forgot to mention I already did the steps there but even if I get no error, the bot is not communicating with the mattermost. So I was hoping for a recent example of working context. I tried the debugging techniques in the https://forum.rasa.com/t/mattermost-channel-debugging/19596 but https://forum.rasa.com/t/mattermost-channel-debugging/19596/18?u=mihai-satmarean this is the result. 

It may be a little thing but I am stuck.

Any hint is appreciated!
